02Arrays/Array/Tester.swift

```diff
@@ -73,7 +140,22 @@ class Tester {
     
     
```

Comment:
does not work when the bestDayToBuy is not the minimum price
Ex. maxProfit(array: [7,3,10,1,2,4]) should return 7